swapi_api/src/modules/mainPage/MainPage.vue

```diff
@@ -1,7 +1,8 @@
 <template>
-    mainpaige
+    <characterDetails/>
 </template>
 
 <script lang="ts" setup>
+import characterDetails from '../characterDetails/CharacterDetails.vue';
```

Comment:
jakim cudem to testowałeś czy działa? Komponent masz z małej, importujesz z dużej?
import CharacterDetails from '../characterDetails/CharacterDetails.vue';